### ISSUE TITLE
Client incompatible with Phoenix 1.2

### ISSUE THREAD

**Issue by weppos** [dnsimple/dnsimple-elixir#110]
```
Failed to use "poison" (version 2.2.0) because
  dnsimple (version 1.1.0) requires ~> 3.1.0
  phoenix (version 1.2.1) requires ~> 1.5 or ~> 2.0
  mix.lock specifies 2.2.0
```

[It was originally planned to use 1.5 or 2.0](https://github.com/dnsimple/dnsimple-elixir/pull/13/files), but then we [ended up using 3.x](https://github.com/dnsimple/dnsimple-elixir/commit/729c81e1bf13b4c95c6ca39caff6f4240e3b0439).

I can't find a ticket/message associated with this latest update. Not sure if it was required by https://github.com/dnsimple/dnsimple-elixir/pull/52

**Comment by weppos** [comment]
Tests are passing:

```
==> dnsimple
Compiling 42 files (.ex)
Generated dnsimple app
................................................................................................................................................

Finished in 10.5 seconds
144 tests, 0 failures
```

**Comment by weppos** [comment]
Closed by #111
